test/Concurrency/async_top_level_no_exit.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %target-typecheck-verify-swift -parse-stdlib
```

Comment:
This will need `-disable-availability-checking` or you'll get an error about using concurrency on older systems.